diff -up dcc-2.3.168/homedir/make-dcc_conf.in.1~ dcc-2.3.168/homedir/make-dcc_conf.in
--- dcc-2.3.168/homedir/make-dcc_conf.in.1~	2023-07-31 23:40:53.769905785 +0200
+++ dcc-2.3.168/homedir/make-dcc_conf.in	2023-07-31 23:41:20.942519737 +0200
@@ -71,7 +71,7 @@ if test -n "$FORCE" -o ! -f "$CUR"; then
     chmod 0644 $CUR$FORCE
     if test -n "@DCCSUID@" -a -z "$NO_SUID" -a -z "$PKG_MAKE"; then
 	set +e		# do not die if the user name is not in the passwd file
-	chown @DCCSUID@ $CUR$FORCE
+	#chown @DCCSUID@ $CUR$FORCE
     fi
     exit
 fi
